src/Steps/Release/CreateChangelog.php

```diff
@@ -245,7 +245,7 @@ protected function getChangelogLibrary(
             // Check if this release has a historic tag.
             $childReleaseName = $childNewReleaseLibrary->getName();
             if (empty($pastComposer['require'][$childReleaseName])) {
-                continue;
+                // continue;
```

Comment:
This fixes "new modules aren't picked up in changelog"

Reply:
Don't leave commented-out code here. Just remove all the dead code.

Reply:
Updated